ts/api/pagopa.ts

```diff
@@ -242,7 +242,9 @@ export type GetWalletsV2UsingGETTExtra = r.IGetApiRequestType<
 >;
 const getWalletsV2: GetWalletsV2UsingGETTExtra = {
   method: "get",
-  url: () => "/v2/wallet",
+  // despite the endpoint is v3, the wallets returned by this API are of type v2
+  // v3 is the same of v2 but in addition it includes Paypal ¯\_(ツ)_/¯
+  url: () => "/v3/wallet",
```

Comment:
If we start to use the `v3/wallet` we should, at least, upgrade the specs in order to add `walletType=generic`, even if it is not handled in the code. Otherwise we would have a decoding error, failing to decode the whole `v3/wallet` response.

Reply:
☝🏻  sure
7c95cff8d55ec6965b702aee7df596ce93f77a47